backend/app/Actions/Visitors/GetVisitorsBounceRateByParameterAction.php

```diff
@@ -101,10 +95,11 @@ public function execute(GetVisitorsBounceRateByParameterRequest $request): GetVi
 
         $response = $collection->map(function ($item) use ($parameter) {
             $bounce_rate = isset($item['bounced_visitors_count']) ? $item['bounced_visitors_count'] / $item['visitors_count'] : 0;
-            return new BounceRateVisitors(
+            return new TableValue(
                 $parameter,
-                $item[$parameter],
-                $bounce_rate
+                is_array($item[$parameter])?$item[$parameter][0]:$item[$parameter],
+                (string)($item['bounced_visitors_count']??0),
+                $bounce_rate * 100
```

Comment:
we have agreed to return from backend values 0 - 1 and transform it to percents on frontend

Reply:
I did as there is for all other table data route